bundles/org.openhab.binding.luftdateninfo/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,105 @@
+<thing:thing-descriptions bindingId="luftdateninfo" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="particulate">
+		<label>Particulate Sensor</label>
+		<description>Sensor to measure Particulate Matter (PM)</description>
+
+		<channels>
+			<channel id="pm25" typeId="pm25-channel" />
+			<channel id="pm100" typeId="pm100-channel" />
+		</channels>
+
+		<config-description>
+			<parameter name="sensorid" type="text" required="true">
+				<label>Sensor ID</label>
+				<description>Sensor ID</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	
+	<thing-type id="conditions">
+		<label>Condition Sensor</label>
+		<description>Sensor to measure Temperature and Humidity conditions</description>
+
+		<channels>
+			<channel id="temperature" typeId="temp-channel" />
+			<channel id="humidity" typeId="hum-channel" />
+			<channel id="pressure" typeId="pressure-channel" />
+			<channel id="pressure-sea" typeId="pressure-sea-channel" />
+		</channels>
+
+		<config-description>
+			<parameter name="sensorid" type="text" required="true">
+				<label>Sensor ID</label>
+				<description>Sensor ID</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="noise">
+		<label>Noise Sensor</label>
+		<description>Sensor to measure noise on location</description>
+
+		<channels>
+			<channel id="noise-eq" typeId="noise-eq-channel" />
+			<channel id="noise-min" typeId="noise-min-channel" />
+			<channel id="noise-max" typeId="noise-max-channel" />
+		</channels>
+
+		<config-description>
+				<parameter name="sensorid" type="text" required="true">
+				<label>Sensor ID</label>
+				<description>Sensor ID</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="pm25-channel">
+		<item-type>Number</item-type>
+		<label>PM 2.5</label>
+		<description>Particulate Matter 2.5</description>
+	</channel-type>
+	<channel-type id="pm100-channel">
+		<item-type>Number</item-type>
+		<label>PM 10.0</label>
+		<description>Particulate Matter 10.0</description>
+	</channel-type>
+	<channel-type id="temp-channel">
+		<item-type>Number</item-type>
+		<label>Temperature</label>
+		<description>Temperature from the selected Sensor ID</description>
+	</channel-type>
+	<channel-type id="hum-channel">
+		<item-type>Number</item-type>
+		<label>Humidity</label>
+		<description>Humidity from the selected Sensor ID</description>
+	</channel-type>
+	<channel-type id="pressure-channel">
+		<item-type>Number</item-type>
+		<label>Atmospheric Pressure</label>
+		<description>Atmospheric Pressure from the selected Sensor ID</description>
+	</channel-type>
+	<channel-type id="pressure-sea-channel">
+		<item-type>Number</item-type>
+		<label>Atmospheric Pressure sea level</label>
```

Comment:
Words in labels should be capitalized. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Corrected